vspace/src/main/java/edu/asu/diging/vspace/web/publicview/ExhibitionSpaceController.java

```diff
@@ -59,6 +64,12 @@ public String space(@PathVariable("id") String id, Model model) {
         if (space.getSpaceStatus() == null || space.getSpaceStatus().equals(SpaceStatus.PUBLISHED)
                 || authenticationFacade.getAuthenticatedUser() != null) {
             IExhibition exhibition = exhibitManager.getStartExhibition();
+            List<ExhibitionAboutPage> aboutPageList = aboutPageManager.findAll();
+            ExhibitionAboutPage exhibitionAboutPage = aboutPageList != null && !aboutPageList.isEmpty() ? aboutPageList.get(0):new ExhibitionAboutPage();
```

Comment:
Let's add a method to the aboutPageManager that returns the about page; once we add multiple languages support this will likely stop working, so putting that logic into its own manager method will make it easier to adjust.

Reply:
Done